### ISSUE TITLE
Please don't block apps on CloudFlare! 

### ISSUE THREAD

**Issue by HappySea666** [uBlockOrigin/uBlock-issues#2009]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] using a new, unmodified browser profile

### Description

When the APP recommended by CloudFlare is installed on the website and uBlock Origin is turned on, the APP becomes invalid!

When uBlock Origin is closed, the APP works normally. 

Cloudflare's web address is cloudflare.com. 

### A specific URL where the issue occurs

Happens to all sites, no specific site. 

### Steps to Reproduce

1. 开启 uBlock Origin 不显示;
2. Turn off uBlock Origin to display normally .

### Expected behavior

Do not affect the normal display of the APP on CloudFlare. 

### Actual behavior

Affect and make the APP not display properly on the website .

### Configuration

<details>

```yaml
uBlock Origin: 1.41.6
Firefox: 97
filterset (summary): 
  network: 97908
  cosmetic: 51278
  scriptlet: 16508
  html: 617
listset (total-discarded, last updated): 
  added: 
    https://gitlab.com/curben/urlhaus-filter/raw/master/urlhaus-filter-online.txt: 7987-0, 21h.18m
    https://raw.githubusercontent.com/cjx82630/cjxlist/master/cjxlist.txt: 516-3, 1d.22h.25m
  default: 
    user-filters: 0-0, never
    ublock-filters: 30852-45, 2d.20h.17m
    ublock-badware: 4023-4, 3d.24m
    ublock-privacy: 198-0, 2d.20h.19m
    ublock-abuse: 77-0, 2d.20h.18m
    ublock-unbreak: 1726-1, 3d.23m
    easylist: 63722-991, 3d.27m
    easyprivacy: 26608-595, 3d.26m
    plowe-0: 3673-2, 11d.32m
    CHN-0: 27928-91, 3d.22m
    urlhaus-1: 8023-7132, 21h.19m
filterset (user): [empty]
trustedset: 
  added: [array of 6 redacted]
  removed: 
    about-scheme
    edge-scheme
    moz-extension-scheme
modifiedUserSettings: 
  contextMenuEnabled: false
  webrtcIPAddressHidden: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1514 ms (selfie)
  maxAssetCacheWait: 1304 ms
```
</details>

**Comment by HappySea666** [comment]
Why is it closed? 



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年2月25日(星期五) 晚上7:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [uBlockOrigin/uBlock-issues] Please don&#39;t block apps on CloudFlare!  (Issue #2009)





 
Closed #2009.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by kolva3** [comment]
report it to [uBlockOrigin/uAssets](https://github.com/uBlockOrigin/uAssets)
